Ansible/Ansible.php

```diff
@@ -15,6 +15,7 @@
 use Asm\Ansible\Command\AnsiblePlaybook;
 use Asm\Ansible\Command\AnsiblePlaybookInterface;
 use Asm\Ansible\Exception\CommandException;
+use Symfony\Component\Process\Process;
```

Comment:
I would be interested in a reproducable scenario for this. I don't see how this can be related to Symfony or this package. But feel free to provide some steps to reproduce it and we can see if that is something we need to fix in Symfony.